gcp_pilot/calendar.py

```diff
@@ -14,11 +14,13 @@
 class Attendee:
     email: str
     optional: bool = False
+    response_status: str = 'needsAction'
```

Comment:
Could you describe into the commit message here why this is needed?

Reply:
It is the same as the `optional`: it is the default for this field: https://googleapis.github.io/google-api-python-client/docs/dyn/calendar_v3.events.html#patch